docs/api-playbook/02-Release Notes/release-notes-v4.md

```diff
@@ -0,0 +1,62 @@
+# V4
+*Product Name: API Playbook*
+*Version: V4*
+*Release Date: September 2022*
+
+### Target Audience
```

Comment:
(test comment for @Sib1-LBH) Should we say target audience here? Or should we delete these versions entirely.